micronota/bfillings/tests/test_infernal.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, micronota development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from tempfile import mkstemp
+from os import getcwd, remove, close
+from unittest import TestCase, main
+
+from skbio.util import get_data_path
+from burrito.util import ApplicationError
+
+from micronota.bfillings.infernal import (
+    CMScan, cmscan_fasta,
+    cmpress_cm)
+
+
+class InfernalTests(TestCase):
+    def setUp(self):
+        self.temp_fd, self.temp_fp = mkstemp()
```

Comment:
yes, to prevent too many unclosed file descriptors.

On Sat, Oct 17, 2015 at 12:10 PM, mortonjt <notifications@github.com> wrote:

> In micronota/bfillings/tests/test_infernal.py
> <https://github.com/biocore/micronota/pull/21#discussion_r42312971>:
>
> > +
> > +from tempfile import mkstemp
> > +from os import getcwd, remove, close
> > +from unittest import TestCase, main
> > +
> > +from skbio.util import get_data_path
> > +from burrito.util import ApplicationError
> > +
> > +from micronota.bfillings.infernal import (
> > +    CMScan, cmscan_fasta,
> > +    cmpress_cm)
> > +
> > +
> > +class InfernalTests(TestCase):
> > +    def setUp(self):
> > +        self.temp_fd, self.temp_fp = mkstemp()
>
> Is this really needed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/micronota/pull/21/files#r42312971>.
>